arangosh/Shell/V8ClientConnection.cpp

```diff
@@ -506,7 +506,7 @@ static void ClientConnection_reconnect(v8::FunctionCallbackInfo<v8::Value> const
   }
 
   V8SecurityFeature& v8security = v8connection->server().getFeature<V8SecurityFeature>();
-  if (!v8security.isAllowedToConnectToEndpoint(isolate, endpoint)) {
+  if (!v8security.isAllowedToConnectToEndpoint(isolate, endpoint, endpoint)) {
     TRI_V8_THROW_EXCEPTION_MESSAGE(TRI_ERROR_FORBIDDEN,
                                    "not allowed to connect to this endpoint");
```

Comment:
Adding my change from #11232 here (will remove it from my PR):
```suggestion
                                   std::string("not allowed to connect to this endpoint")
                                   endpoint);
```